extensions/wikia/ArticleComments/js/ArticleComments.js

```diff
@@ -123,7 +123,8 @@
 					id: commentId,
 					method: 'axEdit',
 					rs: 'ArticleCommentsAjax',
-					useskin: window.skin
+					useskin: window.skin,
+					cb: Date.now()
```

Comment:
Well, geez, that was super easy. This ticket had been bandied back and forth for a while before this PR and was starting to drag. Thanks for the help @macbre, that looks like exactly the way to do it :)